app/src/main/java/com/github/se/orator/ui/overview/SpeakingPublicSpeaking.kt

```diff
@@ -0,0 +1,143 @@
+package com.github.se.orator.ui.overview
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.statusBarsPadding
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.github.se.orator.R
+import com.github.se.orator.ui.navigation.NavigationActions
+import com.github.se.orator.ui.navigation.Screen
+
+
+@Composable
+fun SpeakingPublicSpeaking(navigationActions: NavigationActions) {
+    var occasion by remember { mutableStateOf("") }
+    var demographic by remember { mutableStateOf("") }
+    var main_points by remember { mutableStateOf("") }
+    var feedback_language by remember { mutableStateOf("") }
+
+    androidx.compose.material.Scaffold(
+        modifier = Modifier
+            .fillMaxSize()
+            .testTag("feedbackScreen"),
+        topBar = {
+            androidx.compose.material.TopAppBar(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .statusBarsPadding(),
+                backgroundColor = Color.White,
```

Comment:
The color should adapt to the theme (dark/light) : maybe it should not be hardcoded in the future.